buddyboss/buddyboss-platform
# Fixed - The Visit Group button in Email Notification is not displaying properly

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1570.

### How to test the changes in this Pull Request:

1. Switch to another language like German or translate the string "Visit Group" to a longer word or phrase.
2. Go to any groups and send an invite.
3. Check the email invitation.
4. Notice the misalignment on the button.

### Proof Screenshots or Video
![image](https://user-images.githubusercontent.com/82648504/128903937-831f3d73-49de-467a-a782-34c6d289c610.png)

### Other information:




### Changelog entry

> Adjusted css in email content for Visit group button to be able to handle longer characters.
